Update copyright

jrcamp opened the pull request:
Followup to #1545 

Lots of churn to use 6 spaces in this line because that's what addlicense uses to I changed it throughout:

```
//       http://www.apache.org/licenses/LICENSE-2.0
```

jrcamp commented:
Would be nice to have a CI check that would fail build if copyright is missing from a file.

tigrannajaryan commented:
> Lots of churn to use 6 spaces in this line because that's what addlicense uses to I changed it throughout:
> 
> ```
> //       http://www.apache.org/licenses/LICENSE-2.0
> ```

Not sure I understand why the change. Is it because some files use 6 spaces, some other use a different number of spaces?

jrcamp commented:
> > Lots of churn to use 6 spaces in this line because that's what addlicense uses to I changed it throughout:
> > ```
> > //       http://www.apache.org/licenses/LICENSE-2.0
> > ```
> 
> Not sure I understand why the change. Is it because some files use 6 spaces, some other use a different number of spaces?

Right, a varying number of spaces were used across files so I changed everything to use the format addlicense uses.
